### PR TITLE
Chore: Improve text

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Improve the PR description of the workflow that updates the candid interfaces.

# Changes

* Change in `/workflows/update-ic.yml`.

# Tests

Not necessary.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker how about this?

File: .github/workflows/update-ic.yml [hunk 1]
```diff
@@ -84,11 +84,11 @@ jobs:
           title: 'Update IC Candid Files'
```
Comment (peterpeterparker) [inline .github/workflows/update-ic.yml:84]: Nice, thx! Should we also remove "IC" from the title?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: .github/workflows/update-ic.yml (hunk 1 above)
Comment (lmuntaner) [reply .github/workflows/update-ic.yml]: done!

**Pull request merged**